### PR TITLE
[Merge with openFEC 3919]Fix Candidate profile financial summary page inconsistent data.

### PR DESCRIPTION
**Pull request by fec-jli**
## Summary (required)

- Resolves #[3076](https://github.com/fecgov/fec-cms/issues/3076)]
Reference openFEC PR: [https://github.com/fecgov/openFEC/pull/3919](https://github.com/fecgov/openFEC/pull/3919)

_Include a summary of proposed changes._
We modified three endpoint files:
CandidateTotalsView
CandidateHistoryView
CommitteeHistoryView

This PR will modified get_candidate function to use above updated endpoints.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Candidate profile financial summary page.

## How to test
1) setup both openFEC and fec-cms on local.
2) openFEC: checkout branch: feature/refactor_ofec_totals_candidate_committees_mv
set SQLA_CONN  = dev
3) setup fec-cms point to your local api:
export DATABASE_URL=postgresql://:@/cfdm_cms_test
export FEC_API_URL=http://localhost:5000
FEC_WEB_API_KEY_PRIVATE=xxxxxxxxx
FEC_WEB_API_KEY_PUBLIC=xxxxxxxxx
export FEC_CMS_ENVIRONMENT=LOCAL
export FEC_FEATURE_HOME_MAP=1&&export FEC_FEATURE_HOME_BARCHARTS=1&& export FEC_FEATURE_AGGR_TOTS=1

4) run several test commands
a) npm run test-single 
b) pytest 
c) ./manage.py test
d) or ./manage.py test home.tests
./manage.py test data.tests

5)./manage.py runserver

6) test candidate profile page

(1) from globe search, pick up any candidate to go to candidate profile page.
will display the latest election year data (election=full) for that candidate.
21_cms_globe_search

(2) from all candidates page , search any candidate or click candidate name.
ex: P00003392
you can click different time period. for P00003392, 
Total receipts(election_full=true) => 2014 receipts + 2016 receipts
22_P00003392_summary

3) from election page to find any candidate to test.
ex: (special candidate, election year =2017) 
2018, Georgia, district 6
—>click show all under Results
click: OSSOFF, T. JONATHAN (H8GA06195)
you will see financial summary page. the election drop down list show even number.
23_special_cand

more examples:
P80001571
P00003392  (C00575795,C00431569)
S0NY00188
H8GA06195  (special )   C00630426
H8IN07259
H6PR00082(Puerto Rico)

Notes: 1)The old function **get_candidate** in views.py is commented out for compare  conveniently. after merging this PR, we can create a ticket to remove it.

2)we also found a couple of existing bugs related with this. will have follow up tickets.

### CODE REVIEW

**Comment by johnnyporkchops** [comment]
@jun, aren't there only `FEC_WEB_API_KEY_PRIVATE` and `FEC_WEB_API_KEY_PUBLIC` keys now?
In how to test you have:
```
export FEC_WEB_API_KEY=xxxxxxx
export FEC_WEB_API_KEY_PUBLIC=xxxxxxx
```

File: fec/data/templates/candidates-single.jinja [hunk 1]
```diff
@@ -112,10 +112,7 @@
             aria-controls="panel6"
             href="#section-6">Filings</a>
             <ul>
-              {% if has_raw_filings %}
-                <li><a href="#raw-filings">Raw electronic filings</a></li>
-              {% endif %}
-              <li><a href="#statements-of-candidacy">Statements of candidacy</a></li>
+             <li><a href="#statements-of-candidacy">Statements of candidacy</a></li>
```
Comment (lbeaufort) [inline fec/data/templates/candidates-single.jinja]: @fec-jli it looks like the raw candidate filings were removed here. Should they have been removed?

File: fec/data/templates/partials/candidate/filings-tab.jinja [hunk 2]
```diff
@@ -6,9 +6,6 @@
   <h2 id="section-6-heading">Candidate filings</h2>
   <div class="slab slab--inline slab--neutral u-padding--left u-padding--right">
     <div class="content__section">
```
Comment (lbeaufort) [inline fec/data/templates/partials/candidate/filings-tab.jinja]: Same here - should raw filings have been removed?

File: fec/data/templates/partials/candidate/financial-summary.jinja [hunk 3]
```diff
@@ -14,12 +14,12 @@
     <ul class="list--bulleted">
       {% for committee in committee_groups['P'] | reverse %}
       <li>
-        <a class="t-sans" href="/data/committee/{{ committee.committee_id }}/?cycle={{ committee.related_cycle }}">{{ committee.name }}</a>
+        <a class="t-sans" href="/data/committee/{{ committee.committee_id }}/?cycle={{ committee.related_cycle }}">{{ committee.name }} ({{ committee.committee_id}})</a>
```
Comment (lbeaufort) [inline fec/data/templates/partials/candidate/financial-summary.jinja]: Adding the committee ID here is a nice change

File: fec/data/views.py [hunk 4]
```diff
@@ -115,18 +115,237 @@ def browse_data(request):
     return render(request, 'browse-data.jinja', {'title': 'Browse data', 'parent': 'data'})
 
 
-def get_candidate(candidate_id, cycle, election_full):
-    """
-    Given candidate_id, cycle, election_full, call the API and get the candidate
-    and candidate financial data needed to render the candidate profile page
-    """
+# def get_candidate(candidate_id, cycle, election_full):
```
Comment (lbeaufort) [inline fec/data/views.py]: If we're not using this code, it should be removed.

File: fec/data/templates/candidates-single.jinja (hunk 1 above)
Comment (fec-jli) [reply fec/data/templates/candidates-single.jinja]: Add back. Thanks.

File: fec/data/templates/partials/candidate/filings-tab.jinja (hunk 2 above)
Comment (fec-jli) [reply fec/data/templates/partials/candidate/filings-tab.jinja]: Add back. Thanks.

**Comment by fec-jli** [comment]
@lbeaufort Thanks for your good catch. I add back **has_raw_filings** section. and clean some useless codes. Could you check it ?

**Pull request merged**